flixel/text/FlxText.hx

```diff
@@ -532,12 +532,16 @@ class FlxText extends FlxSprite
 		return Text;
 	}
 	
-	private inline function get_size():Float
+	private inline function get_size():Int
 	{
-		return _defaultFormat.size;
+		#if openfl_legacy
+			return Std.int(_defaultFormat.size);
+		#else
+			return _defaultFormat.size;
```

Comment:
This code seems to trigger even when compiling with legacy (leading to a compiler error: `Null<Float> should be Int`)